src/main/java/org/duckstudy/model/lotto/Lotto.java

```diff
@@ -0,0 +1,42 @@
+package org.duckstudy.model.lotto;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class Lotto {
+
+    public static final int START_INCLUSIVE_NUMBER = 1;
+    public static final int END_EXCLUSIVE_NUMBER = 46;
+    public static final List<Integer> NUMBERS = makeNumbers();
+    public static final int FROM_INDEX = 0;
+    public static final int LOTTO_SIZE = 6;
+
+    public final List<Integer> lotto;
+
+    public Lotto() {
+        this.lotto = Collections.unmodifiableList(makeLotto());
+    }
+
+    private static List<Integer> makeNumbers() {
+        return IntStream.range(START_INCLUSIVE_NUMBER, END_EXCLUSIVE_NUMBER)
+                .boxed()
+                .toList();
+    }
+
+    private List<Integer> makeLotto() {
+        List<Integer> numbers = new ArrayList<>(NUMBERS);
+
+        Collections.shuffle(numbers);
+        numbers = numbers.subList(FROM_INDEX, LOTTO_SIZE);
+
+        Collections.sort(numbers);
+
+        return numbers;
```

Comment:
```suggestion
        Collections.shuffle(NUMBERS);
        return NUMBERS.stream()
                .limit(6)
                .sorted()
                .collect(Collectors.toList());
```
stream을 활용한다면 이렇게 표현될 수 있을 것 같아요. 개인적으로 메소드 내에서 `new ArrayList<>()`를 사용할 수 밖에 없는 로직의 경우 어쩔 수 없지만 가변적인 list를 내부에서 컨트롤했을 때 값 자체가 변경될 확률이 너무 높아서 지양하는 편이에요.

Reply:
그렇군요 !! 불변 리스트로 가변 리스트를 만들더라도 가변 리스트는 언제나 변경이 가능하기 때문에 위험하네요 !
`stream`은 연산시에 원본을 변경하지 않으니 불변 리스트에 stream을 사용하는 것이 좋겠네요!
`불변 리스트 + stream` 조합 기억하겠습니다,, 감사합니다~!!